### PR TITLE
Create music album and genre tasks

### PR DESCRIPTION
**Pull request by MattGomb**
Hey guys,

Here are the requirements for this task:

Create MusicAlbum class in a separate .rb file.

Create Genre class with an association to the Item class (in a separate .rb file).

All MusicAlbum class properties visible in the diagram should be defined and set up in the constructor method.

All Genre class properties visible in the diagram should be defined and set up in the constructor method.

- Implement methods:

add_item method in the Genre class

should take an instance of the Item class as an input
should add the input item to the collection of items
should add self as a property of the item object (by using the correct setter from the item object)
can_be_archived?() in the MusicAlbum class

should override the method from the parent class
should return true if the parent's method returns true AND if on_spotify equals true
otherwise, it should return false

### CODE REVIEW

File: music_album.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class MusicAlbum < Item
+  def initialize(on_spotify: false)
+    super
+    @on_spotify = on_spotify
+  end
+
+  def on_spotify?
+    # Should this be like: "return true if @on_spotify", or just this keyword is enough..?
+    @on_spotify
+  end
```
Comment (jfoyarzo) [inline music_album.rb]: For the MusicAlbum class, I think we only need `on_spotify ` as a property and the only method the class has is the override of `can_be_archived?`, so I think we could remove this method.
image

**Review by jfoyarzo — COMMENTED** [review]
Hey @MattGomb , great job so far! There's only one observation with the method on the music_album.rb file :)

File: music_album.rb (hunk 1 above)
Comment (MattGomb) [reply music_album.rb]: you are right, i removed it!

**Comment by MattGomb** [comment]
@Dadadon  can you do your review please?

**Review by jfoyarzo — APPROVED** [review]

**Review by Dadadon — APPROVED** [review]
Great job! @MattGomb. All the requirements were met. You may merge this pull request.

**Pull request merged**